### PR TITLE
powershell script to call yammer api to bulk revoke tokens

### PR DESCRIPTION
**Pull request by meizhi-2019**
This PR contains 3 files:
1. powershell script which will call Yammer public API to bulk revoke yammer tokens
2. README.md
3. LICENSE which was copied from other projects

### CODE REVIEW

File: bulk-revoke-tokens-script/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+POST: https://www.yammer.com/api/v1/tokens/deletion 
+
+This API will delete all the yammer bearer tokens for the users which provided in the post data "user_ids".
+
+A curl command example to call this API:
+```
+curl -v -d '{"user_ids":[1,2,3]}' -H "Content-Type: application/json" -H "Authorization: Bearer <AAD_Token>" -X POST https://www.yammer.com/api/v1/tokens/deletion
+```
+
+## Work Flow
```
Comment (dligge) [inline bulk-revoke-tokens-script/README.md]: This heading should be "Workflow", one word.

File: bulk-revoke-tokens-script/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Microsoft Corporation
```
Comment (dligge) [inline bulk-revoke-tokens-script/LICENSE]: Let's update the year to 2019.

File: bulk-revoke-tokens-script/Bulk-Revoke-Tokens.ps1 [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    if ($Batch.Count -gt 0)
+    {
+      $Json = ('{{"user_ids": [{0}]}}' -f ($Batch -join ","))
+      RequestDeletion -Json $Json -BearerToken $BearerToken -Api $Api
+      $Batch.Clear()
+    }
+  }
+}
+else {
```
Comment (dligge) [inline bulk-revoke-tokens-script/Bulk-Revoke-Tokens.ps1]: The opening bracket starts on the line after every other conditional statement in this script, so I think we should move it down a line here.

File: bulk-revoke-tokens-script/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+This is a powershell script Sample which will invoke Yammer public API to bulk revoke Yammer legacy token.
+
+## Usage
+```
+Bulk-Revoke-Tokens.ps1 [-CsvPath] <string> [-BearerToken] <string> [-Is3rdPartyAppOnly] <string> [<CommonParameters>]
+```
+This script takes two parameters to run
+* CsvPath: The path of the CSV file which contains "user_id" column
+* BearerToken: AAD token string which will be used to do authentication
+* Is3rdPartyAppOnly: true will only delete tokens for third party applications, false will delate all the tokens
```
Comment (dligge) [inline bulk-revoke-tokens-script/README.md]: Typo: "delate" should be "delete".

File: bulk-revoke-tokens-script/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# bulk-revoke-yammer-token
+This is a powershell script Sample which will invoke Yammer public API to bulk revoke Yammer legacy token.
```
Comment (dligge) [inline bulk-revoke-tokens-script/README.md]: The word "Sample" should be removed here. "Token" should be "tokens".

File: bulk-revoke-tokens-script/README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+```
+This script takes two parameters to run
+* CsvPath: The path of the CSV file which contains "user_id" column
+* BearerToken: AAD token string which will be used to do authentication
+* Is3rdPartyAppOnly: true will only delete tokens for third party applications, false will delate all the tokens
+
+## Yammer API
+POST: https://www.yammer.com/api/v1/tokens/deletion 
+
+This API will delete the yammer bearer tokens for the users which provided in the post data "user_ids". Depending on user's choise, the API will delete all tokens if "false" provided as the value for "Is3rdPartyAppOnly", if user type "true" for "Is3rdPartyAppOnly", only third party application tokens will be deleted.
```
Comment (dligge) [inline bulk-revoke-tokens-script/README.md]: "which provided" should be "which are provided".

File: bulk-revoke-tokens-script/README.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# bulk-revoke-yammer-token
+This is a powershell script which will invoke Yammer public API to bulk revoke Yammer legacy tokens.
+
+## Usage
+```
+Bulk-Revoke-Tokens.ps1 [-CsvPath] <string> [-BearerToken] <string> [-Is3rdPartyAppOnly] <string> [<CommonParameters>]
```
Comment (dligge) [inline bulk-revoke-tokens-script/README.md]: What does "[<CommonParameters>]" refer to?

Comment (meizhi-2019) [reply bulk-revoke-tokens-script/README.md]: This usage was coming from powershell [Get-Help](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/get-help?view=powershell-6)

**Pull request merged**